packages/web/src/pages/sign-on/components/desktop/MetaMaskModal.jsx

```diff
@@ -104,38 +109,41 @@ class MetaMaskModal extends Component {
             />
           </div>
           <div className={styles.actionContainer}>
-            <Tooltip
-              placement='top'
-              overlayClassName={cn(styles.configOverlay, {
-                [styles.hidden]: configured
-              })}
-              title={
-                <span className={styles.configMetaMask}>
-                  {messages.metaMaskConfigure}
-                </span>
-              }
-              getPopupContainer={(trigger) => trigger.parentNode}
-            >
-              <div>
-                <Button
-                  type={configured ? ButtonType.COMMON : ButtonType.DISABLED}
-                  text={messages.continueOption}
-                  onClick={this.onClickContinue}
-                  textClassName={styles.actionButtonText}
-                  className={cn(styles.actionButton, {
-                    [styles.continueButton]: configured
-                  })}
-                />
-              </div>
-            </Tooltip>
+            <div>
+              <Button
+                type={
```

Comment:
do we want to add "disabled" prop as well so it's not clickable? or does stems button do that automatically?